indico/modules/events/registration/client/js/components/EmailPendingInvitationsButton.jsx

```diff
@@ -10,33 +10,55 @@ import emailPreviewURL from 'indico-url:event_registration.api_invitations_remin
 import emailSendURL from 'indico-url:event_registration.api_invitations_reminders_send';
 
 import PropTypes from 'prop-types';
-import React, {useState} from 'react';
+import React, {useEffect, useState} from 'react';
 
 import {Translate} from 'indico/react/i18n';
 
 import EmailPendingInvitations from './EmailPendingInvitations';
 
-export default function EmailPendingInvitationsButton({eventId, regformId, hasPendingInvitations}) {
+const getIds = checkboxes =>
```

Comment:
This already exists as an exported function. Let's move it to a more generic place (e.g. indico/web/client/js/react/util/html.js) and import it instead of duplicating the code.

Reply:
no longer needed anyway after refactoring the list 🙂